src/cart/_structures_from_macros_.h

```diff
@@ -179,17 +179,4499 @@
+	d_iov_t ivs_sync_type;
+	uint32_t ivs_class_id;
+	};
+	struct crt_iv_sync_out {
+	int32_t rc;
+	};
+	extern struct crt_req_format CQF_crt_iv_sync;
+struct crt_ctl_ep_ls_in {
+	crt_group_id_t cel_grp_id;
+	d_rank_t cel_rank;
+	};
+	struct crt_ctl_ep_ls_out {
+	d_iov_t cel_addr_str;
+	int32_t cel_ctx_num;
+	int32_t cel_rc;
+	};
+	extern struct crt_req_format CQF_crt_ctl_ep_ls;
+struct crt_ctl_get_uri_cache_in {
+	crt_group_id_t cel_grp_id;
+	d_rank_t cel_rank;
+	};
+	struct crt_ctl_get_uri_cache_out {
+	struct {
+	uint64_t ca_count;
+	struct crt_grp_cache *ca_arrays;
+	} cguc_grp_cache;
+	int32_t cguc_rc;
+	};
+	extern struct crt_req_format CQF_crt_ctl_get_uri_cache;
+struct crt_ctl_get_host_in {
+	crt_group_id_t cel_grp_id;
+	d_rank_t cel_rank;
+	};
+	struct crt_ctl_get_host_out {
+	d_iov_t cgh_hostname;
+	int32_t cgh_rc;
+	};
+	extern struct crt_req_format CQF_crt_ctl_get_host;
+struct crt_ctl_get_pid_in {
+	crt_group_id_t cel_grp_id;
+	d_rank_t cel_rank;
+	};
+	struct crt_ctl_get_pid_out {
+	int32_t cgp_pid;
+	int32_t cgp_rc;
+	};
+	extern struct crt_req_format CQF_crt_ctl_get_pid;
+struct crt_proto_query_in {
+	d_iov_t pq_ver;
+	int32_t pq_ver_count;
+	uint32_t pq_base_opc;
+	};
+	struct crt_proto_query_out {
+	uint32_t pq_ver;
+	int32_t pq_rc;
+	};
+	extern struct crt_req_format CQF_crt_proto_query;
+struct crt_ctl_fi_attr_set_in {
+	uint32_t fa_fault_id;
+	uint32_t fa_interval;
+	uint64_t fa_max_faults;
+	uint32_t fa_err_code;
+	uint32_t fa_probability_x;
+	uint32_t fa_probability_y;
+	d_string_t fa_argument;
+	};
+	struct crt_ctl_fi_attr_set_out {
+	int32_t fa_ret;
+	};
+	extern struct crt_req_format CQF_crt_ctl_fi_attr_set;
+struct crt_ctl_fi_toggle_in {
+	_Bool op;
+	};
+	struct crt_ctl_fi_toggle_out {
+	int32_t rc;
+	};
+	extern struct crt_req_format CQF_crt_ctl_fi_toggle;
+void crt_req_destroy(struct crt_rpc_priv *rpc_priv);
+static inline _Boolcrt_rpc_cb_customized(struct crt_context *crt_ctx, crt_rpc_t *rpc_pub){
+	return crt_ctx->cc_rpc_cb != ((void *)0);
+}int crt_rpc_priv_alloc(crt_opcode_t opc, struct crt_rpc_priv **priv_allocated, _Bool forward);
+void crt_rpc_priv_free(struct crt_rpc_priv *rpc_priv);
+int crt_rpc_priv_init(struct crt_rpc_priv *rpc_priv, crt_context_t crt_ctx, _Bool srv_flag);
+void crt_rpc_priv_fini(struct crt_rpc_priv *rpc_priv);
+int crt_req_create_internal(crt_context_t crt_ctx, crt_endpoint_t *tgt_ep, crt_opcode_t opc, _Bool forward, crt_rpc_t **req);
+int crt_internal_rpc_register(void);
+int crt_rpc_common_hdlr(struct crt_rpc_priv *rpc_priv);
+int crt_req_send_internal(struct crt_rpc_priv *rpc_priv);
+static inline _Boolcrt_req_timedout(struct crt_rpc_priv *rpc_priv){
+	return (rpc_priv->crp_state == RPC_STATE_REQ_SENT || rpc_priv->crp_state == RPC_STATE_URI_LOOKUP || rpc_priv->crp_state == RPC_STATE_ADDR_LOOKUP || rpc_priv->crp_state == RPC_STATE_TIMEOUT || rpc_priv->crp_state == RPC_STATE_FWD_UNREACH) && !rpc_priv->crp_in_binheap;
+}static inline uint64_tcrt_set_timeout(struct crt_rpc_priv *rpc_priv){
```

Comment:
(style)  space required before the open brace '{'
(style)  space required after that close brace '}'
(style)  storage class should be at the beginning of the declaration